neqo-http3/Cargo.toml

```diff
@@ -32,7 +32,7 @@ strum = { workspace = true}
 url = { workspace = true }
 
 [dev-dependencies]
-criterion = { version = "0.6", default-features = false, features = ["cargo_bench_support"] }
+# criterion = { version = "0.6", default-features = false, features = ["cargo_bench_support"] }
```

Comment:
Remove all together? I can re-introduce it in https://github.com/mozilla/neqo/pull/2890.